src/interactive/Dropdown.jsx

```diff
@@ -133,25 +172,38 @@ class Dropdown extends React.PureComponent {
 					{trigger}
 				</div>
 
-				<div
-					ref={(el) => this.contentRef = el}
-					className={classNames.content}
-					aria-hidden={!isActive}
-				>
-					{content}
-				</div>
+				<Portal isOpened={isActive}>
+					<div
+						ref={(el) => this.contentRef = el}
+						className={classNames.content}
+						aria-hidden={!isActive}
+						style={{
+							left: `${this.state.posX}px`,
+							top: `${this.state.posY}px`,
+							minWidth: `${width}px`,
```

Comment:
Should these both be props and not include `px`? I can imagine wanting to have a separate `minWidth` and `maxWidth`, and for the values possibly to be `auto`.